cde-core/resource/js/cdf-dd.js

```diff
@@ -1881,11 +1881,12 @@ $(function() {
     },
     content: function(data, settings, original) {
       // If it is string assume it is an array.
+      var json;
       if(String == data.constructor) {
-        eval('var json = ' + data);
+        json = JSON.parse(data);
```

Comment:
@jvelasques, what do you think is the best solution? The straightforward approach should be replacing ```'``` with ```"``` before invoking ```parse()```. A bit awkward, but works. 
```
var p = "{'a':'a'}";
p = p.replace(/'/g, '"');
JSON.parse(p);
```